src/Appwrite/Platform/Tasks/ScheduleFunctions.php

```diff
@@ -40,32 +40,39 @@ protected function enqueueResources(Group $pools, Database $dbForConsole): void
 
         $delayedExecutions = []; // Group executions with same delay to share one coroutine
 
-        foreach ($this->schedules as $key => $schedule) {
-            $cron = new CronExpression($schedule['schedule']);
-            $nextDate = $cron->getNextRunDate();
-            $next = DateTime::format($nextDate);
+        foreach ($this->schedules as $scheduleKey => $schedule) {
+            if (CronExpression::isValidExpression($schedule['schedule'])) {
+                $cron = new CronExpression($schedule['schedule']);
+                $nextDate = $cron->getNextRunDate();
+            } else {
+                try {
+                    $nextDate = new \DateTime($schedule['schedule']);
+                    $schedule['delete'] = true;
+                } catch (\Exception) {
+                    Console::error('Failed to parse schedule: ' . $schedule['schedule']);
+                    continue;
+                }
+            }
 
+            $next = DateTime::format($nextDate);
             $currentTick = $next < $timeFrame;
 
             if (!$currentTick) {
                 continue;
             }
 
-            $total++;
-
-            $promiseStart = \time(); // in seconds
-            $executionStart = $nextDate->getTimestamp(); // in seconds
-            $delay = $executionStart - $promiseStart; // Time to wait from now until execution needs to be queued
+            $total += 1;
+            $delay = $nextDate->getTimestamp() - \time(); // Time to wait from now until execution needs to be queued
```

Comment:
In messaigng we do something a bit different:
https://github.com/appwrite/appwrite/blob/feat-scheduled-executions/src/Appwrite/Platform/Tasks/ScheduleMessages.php#L34-L36

Sould we do the same? I think this method runs every minute.. I am worried that with this logic, if you schedule function in 5 mins, it will run 5 times. Maybe.
Also scarred if worker is down, and it gets negative number here, god knows what happens.

I think it's the best bet to copy 99% of code from messaging - messaging scheduling works great.